kingpin/utils.py

```diff
@@ -347,6 +347,30 @@ def convert_json_to_dict(json_file, tokens):
     return decoded
 
 
+def order_dict(obj):
```

Comment:
So a bit of nit-picking but let's be correct here. Python dictionaries don't have a concept of an order. Here's an example:

```
In [1]: {'foo1': 'bar1', 'foo2': 'bar2'} == {'foo2': 'bar2', 'foo1': 'bar1'}
Out[1]: True
```

What does have order is lists. 

```
In [2]: ['a', 'b'] == ['b', 'a']
Out[2]: False
```

So the helper part here that's being leveraged is ordered lists (be they inside of other lists or other dicts). But saying Ordered Dict is just not correct